src/test/java/kitchenpos/dao/OrderLineItemDaoTest.java

```diff
@@ -0,0 +1,151 @@
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("OrderLineItemDao 테스트")
+@SpringBootTest
+@Transactional
+class OrderLineItemDaoTest {
+    @Autowired
+    private OrderLineItemDao orderLineItemDao;
+    @Autowired
+    private MenuDao menuDao;
+    @Autowired
+    private OrderDao orderDao;
+    @Autowired
+    private OrderTableDao orderTableDao;
+
+    private Menu menu;
+    private Order order;
+    private OrderTable orderTable;
+    private OrderLineItem orderLineItem;
+
+    private static Stream<Arguments> saveFailureWhenDbLimit() {
+        return Stream.of(
+                Arguments.of(0L, OrderLineItemFixture.FIRST_FIRST_ORDERLINE.getMenuId()),
+                Arguments.of(null, OrderLineItemFixture.FIRST_FIRST_ORDERLINE.getMenuId()),
+                Arguments.of(OrderLineItemFixture.FIRST_FIRST_ORDERLINE.getOrderId(), 0L),
+                Arguments.of(OrderLineItemFixture.FIRST_FIRST_ORDERLINE.getOrderId(), null)
+        );
+    }
+
+    @BeforeEach
+    void init() {
+        menu = menuDao.save(MenuFixture.후라이드치킨_NO_KEY);
+        orderTable = orderTableDao.save(OrderTableFixture.TABLE_BEFORE_SAVE);
+
+        final Order order = new Order();
+        order.setOrderedTime(LocalDateTime.now());
+        order.setOrderTableId(orderTable.getId());
+        order.setOrderStatus(OrderStatus.COOKING.name());
+        this.order = orderDao.save(order);
+
+        final OrderLineItem orderLineItem = new OrderLineItem();
+        orderLineItem.setMenuId(menu.getId());
+        orderLineItem.setOrderId(this.order.getId());
+        orderLineItem.setQuantity(2);
+        this.orderLineItem = orderLineItemDao.save(orderLineItem);
+    }
+
+    @DisplayName("주문항목 저장 - 실패 - DB제약사항")
+    @CustomParameterizedTest
+    @MethodSource("saveFailureWhenDbLimit")
+    void saveFailureWhenDbLimit(@AggregateWith(OrderLineItemAggregator.class) OrderLineItem orderLineItem) {
+        //given
+        //when
+        //then
+        assertThatThrownBy(() -> orderLineItemDao.save(orderLineItem))
+                .isInstanceOf(DataIntegrityViolationException.class);
+    }
+
+    @DisplayName("주문항목 조회 - 성공 - id 기반 조회")
+    @Test
+    void findById() {
+        //given
+        //when
+        final Optional<OrderLineItem> actual = orderLineItemDao.findById(orderLineItem.getSeq());
+        //then
+        assertThat(actual).isNotEmpty();
+        assertThat(actual.get().getOrderId()).isEqualTo(orderLineItem.getOrderId());
+        assertThat(actual.get().getMenuId()).isEqualTo(orderLineItem.getMenuId());
+        assertThat(actual.get().getQuantity()).isEqualTo(orderLineItem.getQuantity());
```

Comment:
```suggestion
        assertThat(actual.get()).usingRecursiveComparison().ignoringFields("seq").isEqualTo(orderLineItem);
```

저의 경우 이런 객체의 필드값을 비교하는 경우에는 `usingRecursiveComparison()`를 주로 이용해요.
[제 블로그](https://unluckyjung.github.io/testcode/2021/06/27/Recursive-Compare/)를 한번 참고해보셔도 좋겠습니다!

Reply:
오... 정말 좋은 피드백이네요!! 블로그에서 사용법까지! 
감사합니다 :)